fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/sink/FlinkSink.java

```diff
@@ -44,13 +44,13 @@
 import static com.alibaba.fluss.flink.utils.FlinkConversions.toFlussRowType;
 
 /** Flink sink for Fluss. */
-class FlinkSink implements Sink<RowData>, SupportsPreWriteTopology<RowData> {
+class FlinkSink<InputT> implements Sink<RowData>, SupportsPreWriteTopology<InputT> {
```

Comment:
The `FlinkSink` is still not generic, as it bounds to `Sink<RowData>` that only accepts a `DataStream<RowData>` as input.